cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -360,51 +369,56 @@ body {
                 border-bottom: 1px solid @gray-40;
             }
 
+            &-grid {
```

Comment:
The whole `-grid` container we can likely remove, but should be done in another PR.